pcl_ros/include/pcl_ros/pcl_node.hpp

```diff
@@ -113,23 +170,23 @@ class PCLNodelet : public nodelet_topic_tools::NodeletLazy
    * specified jitter. See also @ref latched_indices_ and approximate_sync.
    **/
   bool use_indices_;
-  /** \brief Set to true if the indices topic is latched.
+  /** \brief Set to true if the indices topic has transient_local durability.
    *
    * If use_indices_ is true, the ~input and ~indices topics generally must
    * be synchronised in time. By setting this flag to true, the most recent
    * value from ~indices can be used instead of requiring a synchronised
    * message.
    **/
-  bool latched_indices_;
+  bool latched_indices_;  // TODO(daisukes): should this be renamed to like `transient_local_indices_`?
 
   /** \brief The message filter subscriber for PointCloud2. */
-  message_filters::Subscriber<PointCloud> sub_input_filter_;
+  message_filters::Subscriber<PointCloud2> sub_input_filter_;  // TODO(xxx): type adaptation
```

Comment:
remove TODO, you have one for that already

Reply:
As I mentioned in the previous PR https://github.com/ros-perception/perception_pcl/pull/383#issue-1467428825, I have not implemented type adaptation as the filters do not use type masquerading. So, I put xxx... I can try this, but it will be after this series of PRs and my own ROS2 project migration (may take a month).

Reply:
I'm asking that you remove the TODO since its already documented in the issue tracker to do.

Reply:
I got it